src/multi.jl

```diff
@@ -197,7 +205,7 @@ function populate_convhull_extreme_values(m::PODNonlinearModel, discretization::
 end
 
 """
-    Less memeory & time efficient, but a easier implementation
+    Less memeory & time efficiency, but a easier implementation
 """
 function _populate_convhull_extreme_values(discretization::Dict, ml_indices::Any, λ::Dict, extreme_point_cnt::Int)
```

Comment:
This function _populate_convhull_extreme_values() is not actively used. But it serves the purpose of verifying the populate_convhull_extreme_values() just in case the multi-dimensional scheme in Julia is different.